### PR TITLE
bump kommander chart version

### PR DESCRIPTION
**Pull request by DanielMSchmidt**

### CODE REVIEW

File: addons/kommander/1.1.x/kommander.yaml [hunk 1]
```diff
@@ -0,0 +1,86 @@
+---
+apiVersion: kubeaddons.mesosphere.io/v1beta1
+kind: ClusterAddon
+metadata:
+  name: kommander
+  labels:
+    kubeaddons.mesosphere.io/name: kommander
+  annotations:
+    catalog.kubeaddons.mesosphere.io/addon-revision: "1.0.0-20"
```
Comment (juliangieseke) [inline addons/kommander/1.1.x/kommander.yaml]:
```suggestion
    catalog.kubeaddons.mesosphere.io/addon-revision: "1.1.0-1"
```

File: addons/kommander/1.1.x/kommander.yaml [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+    appversion.kubeaddons.mesosphere.io/kommander: "1.0.0"
```
Comment (juliangieseke) [inline addons/kommander/1.1.x/kommander.yaml]:
```suggestion
    appversion.kubeaddons.mesosphere.io/kommander: "1.1.0"
```

File: addons/kommander/1.1.x/kommander.yaml [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+    - name: gcp
+      enabled: true
+    - name: docker
+      enabled: true
+    - name: none
+      enabled: true
+  chartReference:
+    chart: kommander
+    repo: https://mesosphere.github.io/charts/stable
+    version: 0.4.29
```
Comment (juliangieseke) [inline addons/kommander/1.1.x/kommander.yaml]: given we're not able to support version `1.0.0` of this addon anymore, we should remove that from the repo I think 🤔 we need to come up with a solution which allows us to support multiple versions of Kommander "soon"

File: addons/kommander/1.1.x/kommander.yaml [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+            deploymentAnnotations:
+              secret.reloader.stakater.com/reload: kommander-thanos-client-tls
+
+      kubeaddons-catalog:
+        image:
+          repository: mesosphere/kubeaddons-catalog
+          tag: "v0.8.3"
+          pullPolicy: IfNotPresent
+
+      kommander-ui:
```
Comment (juliangieseke) [inline addons/kommander/1.1.x/kommander.yaml]: do we still need this?

File: addons/kommander/1.1.x/kommander.yaml (hunk 1 above)
Comment (shaneutt) [reply addons/kommander/1.1.x/kommander.yaml]: :heavy_check_mark: the revision should be `${CURRENT_APP_VERSION}-${REV}`, sot his suggestion is :+1:

File: addons/kommander/1.1.x/kommander.yaml (hunk 2 above)
Comment (shaneutt) [reply addons/kommander/1.1.x/kommander.yaml]: This would seem to be an appropriate patch.

File: addons/kommander/1.1.x/kommander.yaml (hunk 3 above)
Comment (shaneutt) [reply addons/kommander/1.1.x/kommander.yaml]: I think we should hold on removing old revisions for now, just for this repo and perhaps discuss this at a sig sync.

Comment (juliangieseke) [reply addons/kommander/1.1.x/kommander.yaml]:
```suggestion
    version: 0.5.0
```

File: addons/kommander/1.1.x/kommander.yaml (hunk 4 above)
Comment (sebbrandt87) [reply addons/kommander/1.1.x/kommander.yaml]: yes? e.g. to set "replicas:" soon (TM)

**Pull request merged**